example/src/components/examples/Buttons.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { Button } from 'react-spark'
+
+const Buttons = () => (<>
+  <h2>Buttons</h2>
+
+  <p>Default:</p>
+  <Button>Default</Button>
+
+  <p>Secondary:</p>
+  <Button variant={'secondary'}>Secondary</Button>
+
+  <p>Secondary:</p>
+  <Button variant={'tertiary'}>Tertiary</Button>
+
+  <p>Disabled:</p>
+  <Button disabled>Disabled</Button>
+
+  <p>Spinner:</p>
+  <Button spinner>Spinner</Button>
```

Comment:
there's a change here to toggle the spinner in the redesign branch, might be worth moving it

Reply:
Nice. I'll merge that branch into this one first.